### PR TITLE
Update bookings-in-outlook.md

### LINKED ISSUE #12787: Line does not read well
The following line does not read very well, and I do not understand it. What does it mean be "then enable"? I *think* I do not need to do anything if I receive an empty response for EwsEnabled, and I do not need to block anything. But it is very unclear.

If the command returns "EwsEnabled:" (empty is default), then enable, but only if need to block "Bookings with", and proceed to Step 2. Otherwise the default values of EwsEnabled is enough to leave "Bookings with me" enabled, no further changes are needed.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7bad8864-5f08-90ae-a6d1-a23e58fc1843
* Version Independent ID: 7bad8864-5f08-90ae-a6d1-a23e58fc1843
* Content: [Bookings with me](https://learn.microsoft.com/en-us/microsoft-365/bookings/bookings-in-outlook?view=o365-worldwide)
* Content Source: [microsoft-365/bookings/bookings-in-outlook.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/bookings/bookings-in-outlook.md)
* Service: **bookings**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
made changes to EWSenabled values to clear ambiguity

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12787

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -88,14 +88,15 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
 
     If the command returns "EwsEnabled: **$true**" then proceed to Step 2.
 
-    If the command returns "EwsEnabled:" (empty is default), then enable, but only if need to block "Bookings with", and proceed to Step 2.
-    Otherwise the default values of EwsEnabled is enough to leave "Bookings with me" enabled, no further changes are needed.
+    If the command returns "EwsEnabled:" (empty is default),  no further changes are needed then proceed to Step 2
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
    If the command returns "EwsEnabled:" (empty is default), no further changes are needed, proceed to Step 2.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@kwekuako

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**